index.html

```diff
@@ -46,6 +46,312 @@ <h1 id="primary" class="flg">
                     <a href="#"><img src="./images/Union.svg" alt="action button"></a>
                 </div>
             </div>
+
+    </section>
+
+    <!-- my recent works section -->
+    <section id="recent-work">
+        <div id="recent-work-text" class="flg">
+            <h1 class="flg">
+                My Recent Works
+            </h1>
+            <img src="./images/Indicator.svg" alt="Indicator" class="flg">
+        </div>
+        <div id="card-row" class="flg">
+            <!--Card One-->
+            <div class="card flg">
+                <div class="card-text">
+                    <h2 class="project-title">
+                        Multi-Post Stories Gain+Glory
+                    </h2>
+                    <div class="tags1 flg">
+                        <div class="inner-tag1 flg">
+                            <h5 class="flg">Ruby on rails</h5>
+                        </div>
+                        <div class="inner-tag1 flg">
+                            <h5 class="flg">css</h5>
+                        </div>
+                        <div class="inner-tag1 flg">
+                            <h5 class="flg">JavScript</h5>
+                        </div>
+                        <div class="inner-tag1 flg">
+                            <h5 class="flg">html</h5>
+                        </div>
```

Comment:
- [ ] Please you should wrap your list of technologies in the `ul` tag and use the `li` tag for each list item to follow best practices and semantics. you can see more on semantics [here](https://www.w3schools.com/html/html5_semantic_elements.asp)